### PR TITLE
Feature/hero path

### PR DESCRIPTION
**Pull request by DrLazer**
helper method to determine hero path

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -16,4 +16,13 @@ def header_image
     # FIXME needs to parse frontpatter
     "media/images/main-header.jpg"
   end
+
+  def hero_image_path(imgpath)
+    if imgpath =~ %r(^/assets/)
+        imgpath
```
Comment (jebw) [inline app/helpers/application_helper.rb:22]: Guessing the linter may complain about the extra whitespace?

File: config/routes.rb [hunk 2]
```diff
@@ -7,5 +7,6 @@
 
   get "/events", to: "pages#events", via: :all
 
+  #get "*page", to: "pages#content", as: :page
```
Comment (jebw) [inline config/routes.rb:10]: This can be dropped I guess

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -6,7 +6,11 @@ def events
     render template: "pages/events"
   end
 
+  def content
```
Comment (jebw) [inline app/controllers/pages_controller.rb:9]: This is now redundant isn't it?

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (jebw) [reply app/helpers/application_helper.rb:22]: Turns out the linter doesn't care :)

Comment (DrLazer) [reply app/helpers/application_helper.rb:22]: lol

**Pull request merged**